lib-ce/emqx_modules/src/emqx_modules.appup.src

```diff
@@ -1,32 +1,41 @@
 %% -*-: erlang -*-
 {VSN,
   [
+    {"4.3.3", [
+        {load_module, emqx_mod_acl_internal, brutal_purge, soft_purge, []}
```

Comment:
Do we need to ensure the metrics are created first instead of load_module directly?

Reply:
good catch!